src/vanna/logger.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import sys
+
+
+def get_logger():
+    """
+    Method to get logger.
+
+    Returns:
+        logging: Logger
+    """
+    return logging.getLogger(__name__)
```

Comment:
If inside vanna/mistral/mistral.py, if we call `.get_logger()`, I would expect to get the logger for `vanna.mistral.mistral`. But I think this always return the logger for "vanna.logger"?

Reply:
I think we should just remove this function entirely. Callers inside this library should just call `import logging; logger = logging.getLogger(__name__)` directly, it is the exact same amount of code.

Application developers should just call `logging.getLogger("vanna")` directly. We should add this to the docs somewhere too.

Reply:
Im not sure I follow. I thought the goal of the PR is to have one standardized logger that is fixed for the entire framework?

Reply:
Have you read the [python logging cookbook](https://docs.python.org/3/howto/logging-cookbook.html)? in the library code, we log to the `vanna.mistral.mistral` logger. Then, a user can either configure that logger directly using  `logging.getLogger("vanna.mistral.mistral")`, or they configure ALL of vanna's loggers at the same time with `logging.getLogger("vanna")`